### ISSUE TITLE
Provide clarity on 32-bit builds

### ISSUE THREAD

**Issue by chekoopa** [nxengine/nxengine-evo#243]
I was going to put the port on my old WinXP PC, but the only 32-bit version available is 2.6.4, which seemingly performs poorly. 
Sources contain no 32-bit project for nx, from which I assume that the support is dropped.

I'd be glad to be mistaken, or get some clarity about the topic. For the while, I'll be using the original engine (advice on 32-bit ports is very welcome).

**Comment by isage** [comment]
Yes, official 32bits build are dropped. You can still build it yourself, you just need to provide 32bit dependencies and add 32bit configuration to project

**Comment by chekoopa** [comment]
> You can still build it yourself, you just need to provide 32bit dependencies and add 32bit configuration to project

Some help or instructions would be appreciated, I've tried doing that but stuck on adding Win32 configuration in nx project.

**Comment by isage** [comment]
A lot has changed in Visual Studio, honestly, and i guess windows sln-projects can be dropped completely, as VS properly supports cmake now.
Someone with windows should really do migration (and build instructions), as i only have windows in a VM and it's a pain to run VS in it.
